4-domain.yml

```diff
@@ -0,0 +1,11 @@
+project:
+  author: Random Guy
+  email: joe@example.com
+  version: 0.99
+  name: Helloer
+  description: 'Da contract dat hallos u'
+  hasstorage: true
+  hasdynamicinvocation: false
+  ispayable: false
+  returntype: ByteArray
+  parameters: ['String', 'Array']
```

Comment:
And different parameters too.

Reply:
This contract takes string (`operation`) and array of additional arguments (`args`) as parameters, so, I thought that it should be 

> ['String', 'Array']

Reply:
Wow, true, I was thinking about `Array` internals here, forgetting about method-params structure.